cmd/snap-bootstrap/triggerwatch/evdev.go

```diff
@@ -88,7 +88,7 @@ func (e *evdevKeyboardInputDevice) probeKeyState() (bool, error) {
 }
 
 func (e *evdevKeyboardInputDevice) WaitForTrigger(ch chan keyEvent) {
-	logger.Noticef("%s: starting wait", e)
+	logger.Noticef("%s: starting wait, %s hold to trigger", e, holdToTrigger)
```

Comment:
"2s hold will trigger"? "2s hold time to trigger" ?

Reply:
```suggestion
	logger.Noticef("%s: starting wait, hold %s to trigger", e, holdToTrigger)
```

Reply:
I suggested "hold 2s to trigger"

Reply:
@bboozzoo I'm a bit confused, are these debugging messages or instructions for the user?

Reply:
Discussed on IRC with @pedronis